jax/_src/test_util.py

```diff
@@ -327,19 +327,24 @@ def _get_device_tags():
     device_tags = {device_under_test()}
   return device_tags
 
-def skip_on_devices(*disabled_devices):
+def skip_on_devices(*disabled_devices, invert_condition=False):
```

Comment:
Nit: I'd probably move the logic into a helper function called by both `skip_on_devices` and `run_on_devices` rather than adding a cryptic `invert_condition` option to the same decorator the tests use

Reply:
I've taken the bulk of `skip_on_devices` and moved it into `_device_filter`.